scripts/migrations/2020-10-miro-migration/chunks.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+"""
+Functions related to gathering migration chunk information
+for creating transfer packages
+"""
+
+import attr
+import click
+import elasticsearch
+import tqdm
+
+from decisions import Decision
+from elastic_helpers import get_local_elastic_client
+
+
+@attr.s
+class Chunk:
+    miro_shard = attr.ib()
+    destination = attr.ib()
+    s3_keys = attr.ib(default=[])
```

Comment:
Beware lists as defaults! Unlike Scala, Python evaluates default arguments *once* and then reuses the same instance everywhere. This can lead to unexpected behaviour if those defaults are mutable e.g.

```pycon
>>> def appender(values=[]):
...     values.append(1)
...     print(values)
...
>>> appender([1, 2, 3])
[1, 2, 3, 1]
>>> appender([1, 2, 4])
[1, 2, 4, 1]
>>> appender()
[1]
>>> appender()
[1, 1]
```

See https://www.attrs.org/en/stable/init.html#defaults